presto-docs/src/main/sphinx/release/release-0.152.3.rst

```diff
@@ -0,0 +1,9 @@
+===============
+Release 0.152.3
+===============
+
+General Changes
+---------------
+
+* Fix incorrect results for grouping sets when task.concurrency is greater than
+  one.
```

Comment:
No need to wrap for one word. The wrapping is to improve readability and does not have a hard margin (same with code). Wrapping here decreases readability.